### DISCUSSION TITLE
1.16-forge will be updated?

### DISCUSSION THREAD

**Discussion by Redacted00** [fnuecke/oc2#89] [Q&A] [answered]
Hey!

Branch [1.16-forge](https://github.com/fnuecke/oc2/tree/1.16-forge) has not been updated for 4 months, when will it continue to update?

**Comment by Dabraleli** [comment, marked as answer]
I don't think 1.16 will get upgates. Atleast for now. Maybe later. Sangar is making 1.18 version ATM.

https://github.com/fnuecke/oc2/issues/85#issuecomment-1003719582_
